maskrcnn_benchmark/modeling/backbone/fbnet_builder.py

```diff
@@ -0,0 +1,822 @@
+        kernel=5,
+        shuffle_type="mid",
+        pw_group=2,
+        se=True,
+        **kwargs
+    ),
+    # layer search 4 (in addition to layer search 3)
+    "ir_k3_sep": lambda C_in, C_out, expansion, stride, **kwargs: IRFBlock(
+        C_in, C_out, expansion, stride, kernel=3, cdw=True, **kwargs
+    ),
+    "ir_k33_e1": lambda C_in, C_out, expansion, stride, **kwargs: IRFBlock(
+        C_in, C_out, 1, stride, kernel=3, cdw=True, **kwargs
+    ),
+    "ir_k33_e3": lambda C_in, C_out, expansion, stride, **kwargs: IRFBlock(
+        C_in, C_out, 3, stride, kernel=3, cdw=True, **kwargs
+    ),
+    "ir_k33_e6": lambda C_in, C_out, expansion, stride, **kwargs: IRFBlock(
+        C_in, C_out, 6, stride, kernel=3, cdw=True, **kwargs
+    ),
+    # layer search 5 (in addition to layer search 4)
+    "ir_k7_e1": lambda C_in, C_out, expansion, stride, **kwargs: IRFBlock(
+        C_in, C_out, 1, stride, kernel=7, **kwargs
+    ),
+    "ir_k7_e3": lambda C_in, C_out, expansion, stride, **kwargs: IRFBlock(
+        C_in, C_out, 3, stride, kernel=7, **kwargs
+    ),
+    "ir_k7_e6": lambda C_in, C_out, expansion, stride, **kwargs: IRFBlock(
+        C_in, C_out, 6, stride, kernel=7, **kwargs
+    ),
+    "ir_k7_sep": lambda C_in, C_out, expansion, stride, **kwargs: IRFBlock(
+        C_in, C_out, expansion, stride, kernel=7, cdw=True, **kwargs
+    ),
+    "ir_k7_sep_e1": lambda C_in, C_out, expansion, stride, **kwargs: IRFBlock(
+        C_in, C_out, 1, stride, kernel=7, cdw=True, **kwargs
+    ),
+    "ir_k7_sep_e3": lambda C_in, C_out, expansion, stride, **kwargs: IRFBlock(
+        C_in, C_out, 3, stride, kernel=7, cdw=True, **kwargs
+    ),
+    "ir_k7_sep_e6": lambda C_in, C_out, expansion, stride, **kwargs: IRFBlock(
+        C_in, C_out, 6, stride, kernel=7, cdw=True, **kwargs
+    ),
+}
+
+
+class Identity(nn.Module):
+    def __init__(self, C_in, C_out, stride):
+        super(Identity, self).__init__()
+        self.conv = (
+            ConvBNRelu(
+                C_in,
+                C_out,
+                kernel=1,
+                stride=stride,
+                pad=0,
+                no_bias=1,
+                use_relu="relu",
+                bn_type="bn",
+            )
+            if C_in != C_out or stride != 1
+            else None
+        )
+
+    def forward(self, x):
+        if self.conv:
+            out = self.conv(x)
+        else:
+            out = x
+        return out
+
+
+class CascadeConv3x3(nn.Sequential):
+    def __init__(self, C_in, C_out, stride):
+        assert stride in [1, 2]
+        ops = [
+            Conv2d(C_in, C_in, 3, stride, 1, bias=False),
+            BatchNorm2d(C_in),
+            nn.ReLU(inplace=True),
+            Conv2d(C_in, C_out, 3, 1, 1, bias=False),
+            BatchNorm2d(C_out),
+        ]
+        super(CascadeConv3x3, self).__init__(*ops)
+        self.res_connect = (stride == 1) and (C_in == C_out)
+
+    def forward(self, x):
+        y = super().forward(x)
```

Comment:
nit: could you add `super(CascadeConv3x3, self).forward(x)` here, for better Py2 compatibility?

Reply:
Updated.